redesign/blog.html

```diff
@@ -10,7 +10,7 @@
     <!-- End Google Tag Manager -->
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="This is our blog, where we talk about our experiences as an agile software development agency">
```

Comment:
The dwyl blog, where you can read about digital product development, agile methodologies, open source and much more